docs/tutorials/optimizers_movingaverage.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>Could you confirm that the <code>-t &lt;path&gt; is necessary</code>  or does it work without specifying that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1057/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
It works without that. Removing it.